src/reducers/computeRenderedColumns.js

```diff
@@ -62,7 +62,7 @@ export default function computeRenderedColumns(state, columnAnchor) {
  * @private
  */
 function calculateRenderedColumnRange(state, columnAnchor) {
-  const bufferColumnCount = 3; // TODO (pradeep): should we calculate this similar to bufferColumnCount ?
+  const bufferColumnCount = 0; // TODO (pradeep): calculate this similar to bufferRowCount
```

Comment:
Our previous behavior was to only render cells inside the viewport. 
Adding a buffer will mean that extra columns are rendered, which doesn't make sense especially for vertical scrolling.

@wcjordan, will need to profile this after #436 is merged
Also if we do end up using this, we'll need to calculate it similar to bufferRowCount